reselpy_randomizer/randomizer.py

```diff
@@ -18,9 +18,11 @@ def json_loader(json_file_spec):
     extensions = ['com', 'net', 'org', 'io', 'tv']
     alphabet = string.ascii_lowercase + string.digits + '!@#$%^&*()_+'
 
+
     @staticmethod
     def get_random_city():
-        return Randomizer.secure_random.choice(Randomizer.cities)
+        city = Randomizer.secure_random.choice(Randomizer.cities)
+        city.map = get_map()
```

Comment:
I can't accept this because there is no implementation of get_map(). Please provide implmentation